docs/getting-started-guides/juju.md

```diff
@@ -33,7 +33,7 @@ and tag the issue with "juju" so we can find them.
 > This documentation focuses on the Juju 2.0 release which will be
 > promoted to stable during the April 2016 release cycle.
 
-To paraphrase, on your local Ubuntu system:  
+To paraphrase, run the following on your local Ubuntu system:  
```

Comment:
"To paraphrase" is weird here. I suggest:

"[Install the Juju client](https://jujucharms.com/docs/2.0/getting-started-general) by running the following command on your local Ubuntu system:"